addons/website/views/snippets/snippets.xml

```diff
@@ -735,7 +735,7 @@
                   data-unit="px"
                   data-save-unit="rem"/>
 
-        <we-row string="Colors">
+        <we-row string="Background">
```

Comment:
I think we changed our minds on that one like 100x :smile: Let's change it one more time :smile: